### DISCUSSION TITLE
okd support on oVirt 4.5

### DISCUSSION THREAD

**Discussion by pdfruth** [okd-project/okd#1253] [Q&A]
okd 4.10 [docs](https://docs.okd.io/4.10/installing/installing_rhv/installing-rhv-user-infra.html) shows installation on oVirt requires oVirt 4.4.

Does okd deployment on oVirt 4.5 work?  Has anyone tried it?
If not, Is there a roadmap to when oVirt 4.5 will be supported?

**Comment by pdfruth** [comment]
I have a partial answer to my own question.

After fixing a small bug in the [create-templates-and-vms.yml](https://github.com/openshift/installer/blob/release-4.10/upi/ovirt/create-templates-and-vms.yml) Ansible playbook, I was able to successfully create templates and VMs on an oVirt 4.5 engine.

The playbook fails with an error message;
```
.
.
Template with name 'worker_rhcos_tpl' and version 'None' in data center 'Default' was not found
.
.
PLAY RECAP *********************************************************************
localhost                  : ok=54   changed=5    unreachable=0    failed=1    skipped=44   rescued=0    ignored=0 
```

This is easily fixed by a one-line code change.
change line 59
from;
`template_name: "{{ item.disks.0.name }}"`
to;
`template_name: "{{ item.template }}"`

I won't be able to actually start all the cluster member vms until I have additional CPU and memory on the oVirt nodes.
But, this is a good early indication that the Ansible playbooks, and associated Python sdk, are compatible with the oVirt 4.5 APIs.

EDIT: I filed an [issue](https://github.com/openshift/installer/issues/5978) for the Ansible playbook

**Comment by fortinj66** [comment]
I've had no issues with 4.10 IPI on ovirt 4.5
